canvg.js

```diff
@@ -231,8 +231,14 @@
 			}
 			else if (window.DOMParser)
 			{
-				var parser = new DOMParser();
-				return parser.parseFromString(xml, 'text/xml');
+				try {
+					var parser = new DOMParser();

```

Comment:
You can define `var parser = new DOMParser()` outside of the try-catch to reuse it.